tests/integration/ambient/baseline_test.go

```diff
@@ -174,17 +173,22 @@ func hboneClient(instance echo.Instance) bool {
 }
 
 func TestServices(t *testing.T) {
-	runAllCallsTest(t, func(t framework.TestContext, src echo.Instance, dst echo.Instance, opt echo.CallOptions) {
+	runAllCallsTest(t, func(t framework.TestContext, src echo.Instance, dst echo.Target, opt echo.CallOptions) {
 		if supportsL7(opt, src, dst) {
 			opt.Check = httpValidator
 		} else {
 			opt.Check = tcpValidator
 		}
 
+		if src.Config().HasSidecar() && t.Settings().AmbientMultiNetwork {
+			t.Skip("https://github.com/istio/istio/issues/57878")
+		}
+
 		if !dst.Config().HasServiceAddressedWaypointProxy() &&
 			!src.Config().HasServiceAddressedWaypointProxy() &&
 			(src.Config().Service != dst.Config().Service) &&
-			!dst.Config().HasSidecar() {
+			!dst.Config().HasSidecar() && opt.Port.Protocol != protocol.GRPC {
```

Comment:
should this change be scoped to only multicluster?

Reply:
good point.  fixed.